packages/web/pages/stake/index.tsx

```diff
@@ -204,7 +211,7 @@ export const Staking: React.FC = observer(() => {
     squadSize,
   ]);
 
-  const isNewUser = usersValidatorsMap.size === 0;
+  const isNewUser = !userHasValPrefs && usersValidatorsMap.size === 0;
```

Comment:
including both here, as there are cases when `userHasValPref` is false but the user has delegated validators